src/terrain/routes/schemas/resource_usage_api.clj

```diff
@@ -76,17 +76,55 @@
           [(optional-key (key keyval)) (val keyval)])))
 
 (defschema CPUHoursTotal
-  {:id              (describe UUID "The UUID assigned to the total")
-   :user_id         (describe UUID "The UUID assigned to the user the total applies to")
+  {:id              (describe String "The UUID assigned to the total")
+   :user_id         (describe String "The UUID assigned to the user the total applies to")
    :username        (describe String "The username of the user the total applies to")
    :total           (describe String "The total number of CPU hours that the user has reserved")
    :effective_start (describe String "The start date for the time range during which the total is applicable")
    :effective_end   (describe String "The end date for the time range during which the total is applicable ")
    :last_modified   (describe String "The date the record in the database was last modified")})
 
+(defschema APIError
+  {:field      (describe String "The top-level JSON field whose generation caused the error to occur")
+   :message    (describe String "The error message")
+   :error_code (describe Integer "The HTTP status code that was generated by the error")})
+
+(defschema Plan
+  {:id          (describe String "The UUID of for the plan")
+   :name        (describe String "The name of the plan")
+   :description (describe String "The description of the plan")})
+
+(defschema QMSUser
+  {:id       (describe String "The UUID QMS assigned the user")
+   :username (describe String "The user's username in QMS, probably the same as in the DE")})
+
+(defschema ResourceType
+  {:id   (describe String "The UUID for the resource type")
+   :name (describe String "The name of the resource type")
+   :unit (describe String "The unit type for the resource")})
+
+(defschema Quota
+  {:id                (describe String "The UUID assigned to the quota")
+   :quota             (describe (maybe Double) "The quota's value")
+   :added_by          (describe (maybe String) "The user that added the quota")
+   :last_modified_by  (describe (maybe String) "When the quota was last modified")
```

Comment:
These fields are going to be removed from the response body soon. Sorry about that; I forgot to mention it in Slack.